pkg/couchdb/couchdb.go

```diff
@@ -47,6 +48,15 @@ func SimpleDatabasePrefix(prefix string) Database {
 	return &simpleDB{prefix}
 }
 
+func rtevent(db Database, evtype, doctype, id, rev string) {
+	realtime.InstanceHub(db.Prefix()).Publish(&realtime.Event{
+		Type:    evtype,
+		DocType: doctype,
+		DocID:   id,
+		DocRev:  rev,
+	})
```

Comment:
The whole document is not sent to the realtime package? How `include_docs` of [the websocket API](https://cozy.github.io/cozy-stack/realtime.html) will work?